apiary.apib

```diff
@@ -3777,11 +3777,41 @@ This request is [asynchronous](#introduction/synchronous-and-asynchronous-calls)
                 + REAL - for number with a decimal point (Synapse, Teradata, Bigquery)
                 + DECIMAL - for numbers (Exasol, Bigquery)
 
-+ Request
++ Request using whereFilters, send as form-data (application/x-www-form-urlencoded)
+    + Headers
+
+            X-StorageApi-Token: your_token
+
+    + Body
+
+            whereFilters%5B0%5D%5Bcolumn%5D=id&whereFilters%5B0%5D%5Boperator%5D=eq&whereFilters%5B0%5D%5Bvalues%5D%5B0%5D=12344
+
++ Request using whereFilters, send as JSON (application/json)
```

Comment:
Ten JSON nemáme nikde otestovaný a ani klient to tak neposílá zatím, ne?

Reply:
jo, ten jsem chtel jeste odstranit, dik za pripomenuti

Reply:
odstraneno